config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+  get "users", to: "users#index"
+  get "users/:user_id/posts/:post_id", to: "users#show_user_post"
+  get "users/:user_id/posts", to: "users#show_user_posts"
+  get "users/:user_id", to: "users#show_user"
```

Comment:
### Optional
You can also write to routes like that.
```

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```

Reply:
Thanks for the feedback, I was unsure about this. I thought we should write down all the routes individually.

Reply:
@jssol It is an easy and clean way to create routes. You can check it with run `rails routes` in the console output will be the same. But still, this is optional.